stac_validator/stac_validator.py

```diff
@@ -98,7 +99,7 @@ def fetch_and_parse_file(self, input_path) -> dict:
     def extensions_val(self, stac_type: str) -> dict:
```

Comment:
We should spell out `val...idation` so that it's not confused with `value` or `valid`